documentation/airport-guides/kiad.md

```diff
@@ -0,0 +1,151 @@
+
+Washington Dulles International Airport is an international airport 26 miles west of downtown Washington. Dulles handles 90% of all international traffic within the Baltimore-Washington region, with 67,000+ passengers passing through its airport daily with routes to/from 116 destinations (<a href="https://www.mwaa.com/sites/default/files/12-19_ats_2.10.20.pdf" target="_blank">December 2019</a>)
+
+Dulles is a hub for United Airlines.
+
+### Airspace: Shenandoah Area (SHD)
+The Shenandoah Area is the part of the Potomac PCT which mainly covers IAD in addition to KFDK and KHEF. The airspace is relatively large, so <a href="https://i.ibb.co/bQSXRmW/image.png" target="_blank">here is a diagram</a>] of the airspace that MTV owns.
+
+### Airports of Interest
+
+#### KIAD/IAD - Washington Dulles International Airport (Primary Airport)
+2019: 24.8m+ passengers
+2019: 308,159 aircraft movements
+
+More information above under *About this facility*.
+
+#### KDCA/DCA - Ronald Reagan Washington National Airport
+DCA is an airport just 3 miles south of downtown Washington. It covers 861 hectares, acting as a fortress hub for American Airlines. The airport is known by many pilots for its River Visual procedure for runway 19 which requires at least 3 statute miles of visibility. It uses the Potomac River and various bridges including the Roosevelt Bridge, Memorial Bridge, Rochambeau Bridge and more.
+
+Mount Vernon (MTV) mainly covers DCA and central portions of the PCT.
+
+#### KBWI/BWI - Baltimore-Washington International Airport
+Baltimore/Washington International Thurgood Marshall Airport is 9 miles south of downtown Baltimore and 32 miles northeast of Washington. BWI handles the most passengers out of all the three major airports within the Baltimore-Washington area. The airport handled 248,585 aircraft movements in 2015-2016; the majority of those flights being operated by Southwest Airlines, which has a fortress hub here.
+
+Chesapeake (CHP) covers Baltimore and the northern area of the PCT.
+
+#### KRIC/RIC - Richmond International Airport
+Richmond International Airport is the smallest of the four major airports within the Potomac Consolidated TRACON, 7 miles southeast of downtown Richmond. During 2015-2016 it handled 3,818,759 passengers across 10,073 aircraft movements.
+
+James River (JRV) covers Richmond and the southern area of the PCT.
+
+### Instrument Procedures
+The instrument procedures are as of AIRAC 2006.
+
+##### Departures
+Dulles has 10 Standard Instrument Departures (SIDs) which are all available in openScope. The only departure which is not assigned automatically is the Capital One (CPTAL1) departure as it is used much less frequently than the others.
+
+
+|Departure Gate|Altitude (ft)|Notes|Handoff To|
```

Comment:
The use of departure gates here as opposed to SIDs is a bit confusing, it could be good to add another column for the corresponding SIDs

Reply:
I extrapolated this out of PCT's VATSIM SOP. I think they do it this way since multiple SIDs have the same gates but I do see how this can be confusing. I'll work to add a column at the left.

Reply:
This looks to cover it all

|SID|Departure Gate(s)|Altitude (ft)|Notes|Handoff To|
|---|---|---|---|---|
|JCOBY#|All|10,000|Direct RIGNZ to join SID|MTV - KRANT|
|CPTAL#|SWANN/AGARD/PALEO/COLIN|10,000|Vector through C-Gate (heading 100-120)|MTV - KRANT|
|WOOLY#|All|11,000|Direct RAZZA to join SID or vector towards RAZZA if vectors WOOLY|CHP|
|JERES#, MCRAY#|All|11,000|On course|CHP - BUFFR|
|BUNZZ#, CPTAL#, RNLDI#|LDN/OTTTO/RAMAY|10,000 or lower filed cruise|On course|MTV - LURAY|
|JDUBB#, CLTCH#, CPTAL#|JDUBB#/CLTCH#: All; CPTAL#: GVE|10,000 or lower filed cruise|On course|MTV - FLUKY|

Reply:
Perfect :+1: